### ISSUE TITLE
Reddit videos are not downloaded the highest quality

### ISSUE THREAD

**Issue by Komnenoi** [yt-dlp/yt-dlp#5224]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting a broken site
- [X] I've verified that I'm running yt-dlp version **2022.10.04** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

Finland

### Provide a description that is worded well enough to be understood

Reddit videos do not seem to be downloading at their highest quality at the moment. Using yt-dlp seems to only download a 720p version while the reddit API shows that there is a 1080p version which can also be downloaded using a third party site such as https://viddit.red/

Video in question that the download was attempted on: https://www.reddit.com/r/UkraineWarVideoReport/comments/xwm9wm/russian_draftees_exasperated_abandoned_left_in/ (Does happen with others as well)

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
C:\Users\Leijona\Downloads\youtubedl>yt-dlp -vU --embed-thumbnail --write-thumbnail --add-metadata https://www.reddit.com/r/UkraineWarVideoReport/comments/xwm9wm/russian_draftees_exasperated_abandoned_left_in/
[debug] Command-line config: ['-vU', '--embed-thumbnail', '--write-thumbnail', '--add-metadata', 'https://www.reddit.com/r/UkraineWarVideoReport/comments/xwm9wm/russian_draftees_exasperated_abandoned_left_in/']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version 2022.10.04 [4e0511f] (win32_exe)
[debug] Python 3.8.10 (CPython 64bit) - Windows-10-10.0.19044-SP0
[debug] Checking exe version: ffmpeg -bsfs
[debug] Checking exe version: ffprobe -bsfs
[debug] exe versions: ffmpeg 4.3.2-2021-02-20-full_build-www.gyan.dev, ffprobe 4.3.2-2021-02-20-full_build-www.gyan.dev
[debug] Optional libraries: Cryptodome-3.15.0, brotli-1.0.9, certifi-2022.09.24, mutagen-1.45.1, sqlite3-2.6.0, websockets-10.3
[debug] Proxy map: {}
[debug] Loaded 1690 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Latest version: 2022.10.04, Current version: 2022.10.04
yt-dlp is up to date (2022.10.04)
[debug] [Reddit] Extracting URL: https://www.reddit.com/r/UkraineWarVideoReport/comments/xwm9wm/russian_draftees_exasperated_abandoned_left_in/
[Reddit] xwm9wm: Downloading JSON metadata
[Reddit] xwm9wm: Downloading m3u8 information
[Reddit] xwm9wm: Downloading MPD manifest
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, hdr:12(7), vcodec:vp9.2(10), channels, acodec, filesize, fs_approx, tbr, vbr, abr, asr, proto, vext, aext, hasaud, source, id
[debug] Default format spec: bestvideo*+bestaudio/best
[info] fvd5ww17x1s91: Downloading 1 format(s): dash-video_2403695+dash-audio_0_131773
[info] Downloading video thumbnail 4 ...
[info] Writing video thumbnail 4 to: Russian draftees exasperated - abandoned, left in the open, they have to buy their food, [fvd5ww17x1s91].png
[debug] Invoking http downloader on "https://v.redd.it/fvd5ww17x1s91/DASH_720.mp4"
[debug] File locking is not supported. Proceeding without locking
[download] Destination: Russian draftees exasperated - abandoned, left in the open, they have to buy their food, [fvd5ww17x1s91].fdash-video_2403695.mp4
[download] 100% of   34.47MiB in 00:00:03 at 10.42MiB/s
[debug] Invoking http downloader on "https://v.redd.it/fvd5ww17x1s91/DASH_audio.mp4"
[download] Destination: Russian draftees exasperated - abandoned, left in the open, they have to buy their food, [fvd5ww17x1s91].fdash-audio_0_131773.m4a
[download] 100% of    1.89MiB in 00:00:00 at 7.16MiB/s
[Merger] Merging formats into "Russian draftees exasperated - abandoned, left in the open, they have to buy their food, [fvd5ww17x1s91].mp4"
[debug] ffmpeg command line: ffmpeg -y -loglevel "repeat+info" -i "file:Russian draftees exasperated - abandoned, left in the open, they have to buy their food, [fvd5ww17x1s91].fdash-video_2403695.mp4" -i "file:Russian draftees exasperated - abandoned, left in the open, they have to buy their food, [fvd5ww17x1s91].fdash-audio_0_131773.m4a" -c copy -map "0:v:0" -map "1:a:0" -movflags "+faststart" "file:Russian draftees exasperated - abandoned, left in the open, they have to buy their food, [fvd5ww17x1s91].temp.mp4"
Deleting original file Russian draftees exasperated - abandoned, left in the open, they have to buy their food, [fvd5ww17x1s91].fdash-audio_0_131773.m4a (pass -k to keep)
Deleting original file Russian draftees exasperated - abandoned, left in the open, they have to buy their food, [fvd5ww17x1s91].fdash-video_2403695.mp4 (pass -k to keep)
[Metadata] Adding metadata to "Russian draftees exasperated - abandoned, left in the open, they have to buy their food, [fvd5ww17x1s91].mp4"
[debug] ffmpeg command line: ffmpeg -y -loglevel "repeat+info" -i "file:Russian draftees exasperated - abandoned, left in the open, they have to buy their food, [fvd5ww17x1s91].mp4" -map 0 -dn -ignore_unknown -c copy -write_id3v1 1 -metadata "title=Russian draftees exasperated - abandoned, left in the open, they have to buy their food," -metadata "date=20221005" -metadata "purl=https://www.reddit.com/r/UkraineWarVideoReport/comments/xwm9wm/russian_draftees_exasperated_abandoned_left_in/" -metadata "comment=https://www.reddit.com/r/UkraineWarVideoReport/comments/xwm9wm/russian_draftees_exasperated_abandoned_left_in/" -metadata "artist=antrophist" -movflags "+faststart" "file:Russian draftees exasperated - abandoned, left in the open, they have to buy their food, [fvd5ww17x1s91].temp.mp4"
[EmbedThumbnail] mutagen: Adding thumbnail to "Russian draftees exasperated - abandoned, left in the open, they have to buy their food, [fvd5ww17x1s91].mp4"
```

**Comment by denaunistic** [comment]
This seems to be fixed with #5165

**Comment by Komnenoi** [comment]
> This seems to be fixed with #5165

If so, that's good and this can be closed. I did try searching but must have missed it as not used to the github titles and stuff. Should have looked closer. Thanks :)
